graylog2-web-interface/src/@types/graylog-web-plugin/index.d.ts

```diff
@@ -73,7 +73,11 @@ interface PluginCloud {
     onCreate: (formData: { [name: string ]: string }) => { [name: string]: string };
   };
 }
-
+interface InputConfiguration {
+  type: string;
+  component: React.ComponentType<{}> | null;
```

Comment:
This component should be defined in each existing input configuration, so we don't need to allow `null`:
```suggestion
  component: React.ComponentType<{}>;
```